surfer/viz.py

```diff
@@ -30,6 +32,12 @@
 logger = logging.getLogger('surfer')
 
 
+# Monkey-patch for Mayavi 4.5: filters are missing a point_data attribute that
+# Threshold accesses on initialization
+if LooseVersion(mayavi.__version__) <= LooseVersion('4.5.0'):
+    mayavi.filters.api.Threshold._get_data_range = lambda x: []
```

Comment:
This fixes it – maybe not ideal to modify the class but acceptable for this purpose since it does not seem to have side-effects?